xml/ha_migration.xml

```diff
@@ -554,6 +565,20 @@
      </tbody>
     </tgroup>
    </informaltable>
+
+   <!--taroth 2020-04-27: the following note is copied from
+    https://github.com/SUSE/doc-sle/edit/maintenance/SLE15SP1/xml/sle_update_upgrading.xml,
+    ll. 57-70 and adjusted to the HA product-->
+   <note>
+    <title>Skipping Service Packs</title>
+    <para>
+     Consecutively installing all Service Packs is the easiest upgrade path.
```

Comment:
```suggestion
     The easiest upgrade path is consecutively installing all service packs.
```

Reply:
We do not seem to be consistently capitalizing "Service Pack(s)" everywhere, so I suggest dropping it.

As a minimum, perhaps, capitalizing only the first usage in a particular book.